server/internal/store/collection.go

```diff
@@ -0,0 +1,159 @@
+package store
+
+import (
+	"fmt"
+	"strings"
+
+	"gorm.io/gorm"
+)
+
+// CollectionStatus represents the status of a collection.
+type CollectionStatus string
+
+// ExpiresAfterAnchor represents the anchor for the expiration time.
+type ExpiresAfterAnchor string
+
+const (
+	// CollectionStatusExpired represents the expired status.
+	CollectionStatusExpired CollectionStatus = "expired"
```

Comment:
Question: Who will update the status? We don't need to implement now, but just curious. (Also when we implement, it's better to have a separate deployment to run background processing.)

Reply:
For expiration, we need to handle it by checking those ExpiresAfter or ExpiresAt timer. For in progress and completed,  I think we will handle those after implementing the APIs related to adding files into the collection.